app/views/directives/osc-image-summary.html

```diff
@@ -4,10 +4,14 @@
   <div ng-show="resource | annotation:'provider'">Provider: {{ resource | annotation:'provider' }}</div>
   <div ng-show="resource.metadata.namespace">Namespace: {{ resource.metadata.namespace }}</div>
 </div>
-<div class="resource-description"
+<div class="resource-description pad-bottom-none"
      ng-bind-html="resource | description | linky"
```

Comment:
We also show the sample repository below the git input. I think this might cause confusion? It's not clear in the description that this is just a sample and you can pick any repo.